mlflow/projects/databricks.py

```diff
@@ -227,10 +227,13 @@ def run_databricks(uri, entry_point, version, parameters, experiment_id, cluster
     if cluster_spec is None:
         raise ExecutionException("Cluster spec must be provided when launching MLflow project runs "
                                  "on Databricks.")
-
-    # Fetch the project into work_dir & validate parameters
-    work_dir = _fetch_project(uri=uri, use_temp_cwd=True, version=version,
-                              git_username=git_username, git_password=git_password)
+    # Fetch the project into temporary work_dir & validate parameters
+    work_dir = _fetch_project(
+        uri=uri, force_tempdir=True, version=version, git_username=git_username,
+        git_password=git_password)
+    mlruns_dir = os.path.join(work_dir, "mlruns")
+    if os.path.exists(mlruns_dir):
+        shutil.rmtree(mlruns_dir)
```

Comment:
Is there a test for this behavior in the databricks-side tests?

Reply:
There wasn't, just added one!